VisualPinball.Unity/VisualPinball.Unity.Editor/Layers/LayersEditor.cs

```diff
@@ -0,0 +1,65 @@
+using UnityEditor;
+using UnityEditor.IMGUI.Controls;
+using UnityEngine;
+
+namespace VisualPinball.Unity.Editor.Layers
+{
+
+	/// <summary>
+	/// This Editor will handle all Layers management as VPX does.
+	/// It's using a custom LayersTreeView
+	/// </summary>
+	public class LayersEditor : EditorWindow
+	{
+		private TreeViewState _treeViewState;
+		private TreeView _treeView;
+
+
+		[MenuItem("Visual Pinball/Layers Manager", false, 101)]

```

Comment:
I would put it into `Window/Visual Pinball/Layer Manager`

Reply:
Hmm, or not, on second thought. It's just that editor windows are usually there, but I see that t he Visual Pinball main menu is kind of empty anyway, so it doesn't hurt there. I'll let you decide!

Reply:
I thought the plan was to have all vpe related menus in the same place as the vpx importer.  
It could be that or spread in existing unity menus but I think we should stick to the same vpe exposure.

Reply:
Okay for me!